### ISSUE TITLE
Middle pointer click uses primary instead of secondary selection (clipboard)

### ISSUE THREAD

**Issue by sumpfralle** [vector-im/riot-web#14430]
### Description

The riot web client seems to override the window manager's handling of *selections* (primary, secondary, clipboard - see [xsel manpage](https://manpages.debian.org/buster/xsel/xsel.1x.en.html)).

The default action of the middle pointer click event is *paste from the secondary selection*. Instead riot overrides this behavior and implements *paste from the primary selection*.

### Steps to reproduce

1. select some text (e.g. in a terminal or a web browser) and copy it to the clipboard (e.g. via the context menu or `CTRL-C`)
2. select some text with the pointer, but do **not** copy it to the clipboard (e.g.: just select it with a double-click on a word)
3. switch to the riot web client
4. press the middle pointer button
5. the content selected in step (1) is pasted into the message input field

What I expected instead: the content selected in step (2) being pasted.

### Details

The window manager handles three selections: *primary*, *secondary* and *clipboard*.

riot seems to override the *middle pointer click* with *paste from primary*. All other applications leave the middle pointer click to the window manager (or some other layer). By default this event triggers a *paste from secondary*.

The current behavior is surprising and may lead to accidental information exposure, e.g. passwords recently acquired from the local password management application being entered (but not yet submitted) into riot's message input field.

Instead the riot client should let the desktop environment handle the middle pointer click event.

### Workaround

1. middle pointer click in a separate text editor (e.g. gedit)
2. select the pasted text
3. copy the selection to the primary clipboard
4. perform a middle pointer click in riot's message input field

This is especially relevant for content copied from applications that do not support *copy to clipboard* directly (e.g. some terminals or visualization application).

### Version information

- **Platform**: web browser and *revolt*

For the web app:

- **Browser**: Firefox 68, Chromium 83
- **OS**: Linux (Debian testing)
- **URL**: https://klax.systemausfall.org; v1.6.8

**Comment by t3chguy** [comment]
riot-web is a webapp and any overriding would be done by the browser running it, whatever revolt is using in your case.

Riot does no such overriding for any clipboard paste events.
The web clipboard spec doesn't even have support for multiple selections: https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/readText (and Riot doesn't use it as it has no reason to do custom clipboard reading)

**Comment by sumpfralle** [comment]
I feel stupid now :(

Yes, indeed it happens only in *revolt*, not in a separate browser. My memory failed me here ...

Sorry for the noise.

**Comment by Marenz** [comment]
The issue might still be one that can be solved here though. 

I looked through various text field inputs and found that it works for all of them except the chat input.
The difference: All of the ones that work are `<input..>` based elements, whereas the text input is a `<div>` field.
I tried setting up a local element-web to change that div to input to try and test my theory. 

Unfortunately so far I failed to set it up to test. Maybe someone here can?
